src/pages/guides/sellers/app-version.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Specify the version
+---
+
+# Specify the version
+
+After you [create the app entry](app-create.md), the next step is to specify and submit a new version. Extensions can have multiple versions, including the first version submitted.
```

Comment:
I believe this section is for **apps** ... if so, then consider scanning for the word `extension` and changing it to `app`.